facebook.com - automated adds icon invert fix

Myshor opened the pull request:
Removed .sx_f77c61
Added ._81u_ .img
Maybe it will be more stable with invert. 😃

Myshor commented:
Two more commits:
1. stories dark icons rework
2. removed invert for .-op i - marketplace gallery previous/next arrows

Myshor commented:
Added invert for ._3ffs li - circles inside gallery(marketplace)

Myshor commented:
added invert: #profile_intro_card_bio i
removed invert: .sx_0f440a

Myshor commented:
Added invert button i._3-8_ - icons to write in messenger in your page community buttons.
Removed ._3qb- .img - as this started to work fine without additional invert

Myshor commented:
.editPhoto i._3-8_ added to invert

Myshor commented:
Added invert for #event_tabs i

Myshor commented:
icons to write in messenger in your page community buttons made more specific as it started to invert other icons back to black.
button[type="submit"] i._3-8_=> .uiList button[type="submit"] i._3-8_

Myshor commented:
Removed #event_tabs span i from invert.

Myshor commented:
![image](https://user-images.githubusercontent.com/56877029/74593018-7da2f680-5027-11ea-942a-4055fc86b9b5.png)
event transparency dialog icons fix

Myshor commented:
invert for event questions added
![image](https://user-images.githubusercontent.com/56877029/74593209-2aca3e80-5029-11ea-8726-18189bf7b4f7.png)

Myshor commented:
Question mark icon must remain black... fix inverts all icons in events. :(

Myshor commented:
Fix for question mark icon without other icons invert. (.sx_8008e8)

Myshor commented:
added invert for next/previous
![image](https://user-images.githubusercontent.com/56877029/74612388-499e0300-5105-11ea-8454-6bb0815050a1.png)

alexanderby commented:
Hi @Myshor, if you are using INVERT for things other from images and icons, could you please explain how to see this problems? It's better to see what is the problem and try to fix them, because after the possible fix the inversion will keep inverting dark content into white.

Myshor commented:
in this requests all was icons and there was no invert for texts.
Do you want screens for all differences here? ;)

Myshor commented:
I make review for all changes and finalize what is really needed here or not. Now I'm in progress.

Myshor commented:
Removed whole fixes for instastories, all icons works fine without invert and CSS fixes.

For the rest of fixes:
._81u_ .img
![image](https://user-images.githubusercontent.com/56877029/74764042-3dd74b80-5281-11ea-892b-ec1d117e2c0a.png)
._3ffs li
![image](https://user-images.githubusercontent.com/56877029/74764153-8262e700-5281-11ea-9c30-21317ae8659e.png)
#profile_intro_card_bio i
![image](https://user-images.githubusercontent.com/56877029/74764445-04eba680-5282-11ea-926e-331d4157f8c3.png)
.uiList button[type="submit"] i._3-8_
![image](https://user-images.githubusercontent.com/56877029/74764680-6e6bb500-5282-11ea-9365-b1bdb522d52a.png)
.editPhoto i._3-8_
![image](https://user-images.githubusercontent.com/56877029/74764777-a672f800-5282-11ea-9681-5afe502a39e1.png)
.sx_08856a
.sx_ac12f7
![image](https://user-images.githubusercontent.com/56877029/74765937-b8559a80-5284-11ea-8ab3-58d00d519b3c.png)
.sx_a1a9dd
![image](https://user-images.githubusercontent.com/56877029/74766766-facba700-5285-11ea-8729-4d70088a81bb.png)
.sp_UJVjdRRsSGX
![image](https://user-images.githubusercontent.com/56877029/74766921-36ff0780-5286-11ea-967b-69d96443710a.png)

Myshor commented:
Added fix for X in theatre mode
Removed old: .xlt from invert
![image](https://user-images.githubusercontent.com/56877029/75144131-a5b0ea80-56f6-11ea-848b-891103580f32.png)

Myshor commented:
Next/previous class change fix:
.sp_UJVjdRRsSGX => .sp_hk4DJV_EEeW

Myshor commented:
Remove ._207_ fix
![image](https://user-images.githubusercontent.com/56877029/75385852-e3716700-58e0-11ea-993f-7e5f50ae46c9.png)

Myshor commented:
Added #u_fetchstream_1_u i._3-8_
The same icon as shown in previous image.
